src/parser/syntax.rs

```diff
@@ -85,61 +89,125 @@ pub fn parse_symbol_limbs<F: LurkField>() -> impl Fn(Span<'_>) -> ParseResult<'_
     }
 }
 
-pub fn parse_absolute_symbol<F: LurkField>() -> impl Fn(Span<'_>) -> ParseResult<'_, F, Symbol> {
+fn intern_path<'a, F: LurkField>(
+    state: Rc<RefCell<State>>,
+    upto: LocatedSpan<&'a str>,
+    path: &[String],
+    keyword: Option<bool>,
+    create_unknown_packages: bool,
```

Comment:
When would `create_unknown_packages` be `true`? Elsewhere you said that packages are required for reading. I'm just trying to get a handle on what the rules are, as currently implemented.

Reply:
This option is mostly for testing, when we want interning not to fail when a package is not found